apturicovid/welcome/WelcomeVC.swift

```diff
@@ -16,7 +16,7 @@ class WelcomeVC: BaseViewController {
     let langViews = Language.allCases.map{ LanguageView.create($0) }
     
     //MARK: Text attributes:
-    let linkFont = UIFont.systemFont(ofSize: 14, weight: .medium)
+    var linkFont:UIFont { UIFont.preferredFont(forTextStyle: .body)}
```

Comment:
why var? Is it being mutated somewhere?

Also - I'd prefer getter body to be on a new line like
```
var linkFont:UIFont { 
    UIFont.preferredFont(forTextStyle: .body)
}
```

Reply:
It’s a computed property.